docs/tinlake/overview/deployments.md

```diff
@@ -7,4 +7,5 @@ redirect_from:
   - /tinlake/
 ---
 
-contractgs
+## Tinlake Deployments
```

Comment:
I would actually move this paragraph into the Introduction (as a paragraph, not a section

Reply:
Let's leave it like that and expand on it later